webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -492,6 +493,19 @@ class NotificationsTab extends React.PureComponent<Props, State> {
         this.props.closeModal();
     };
 
+    Input = (props: InputProps) => {
+        const ariaProps = {
+            'aria-labelledby': 'settingTitle',
+            'aria-describedby': 'extraInfo',
+        };
+
+        return (
+            <components.Input
+                {...props}
+                {...ariaProps}
+            />);
+    };
+
```

Comment:
instead of this being a function inside of the class, can you create a component in the same file that way it wont be recreated on each render